container-engine-lib/lib/backend_impls/kubernetes/object_attributes_provider/logs_collector_object_attributes_provider.go

```diff
@@ -0,0 +1,98 @@
+package object_attributes_provider
+
+import (
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_impls/kubernetes/object_attributes_provider/kubernetes_annotation_key"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_impls/kubernetes/object_attributes_provider/kubernetes_annotation_value"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_impls/kubernetes/object_attributes_provider/kubernetes_label_key"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_impls/kubernetes/object_attributes_provider/kubernetes_label_value"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_impls/kubernetes/object_attributes_provider/label_value_consts"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_interface/objects/logs_collector"
+	"github.com/kurtosis-tech/stacktrace"
+)
+
+const (
+	logsCollectorNamePrefix       = "kurtosis-logs-collector"
+	logsCollectorConfigNamePrefix = "kurtosis-logs-collector-config"
+)
+
+type KubernetesLogsCollectorObjectAttributesProvider interface {
+	ForLogsCollectorDaemonSet() (KubernetesObjectAttributes, error)
+
+	ForLogsCollectorNamespace() (KubernetesObjectAttributes, error)
+
+	ForLogsCollectorConfigMap() (KubernetesObjectAttributes, error)
+}
+
+func GetKubernetesLogsCollectorObjectAttributesProvider(logsCollectorGuid logs_collector.LogsCollectorGuid) KubernetesLogsCollectorObjectAttributesProvider {
+	return newKubernetesLogsCollectorObjectAttributesProvider(logsCollectorGuid)
+}
+
+type kubernetesLogsCollectorObjectAttributesProviderImpl struct {
+	logsCollectorGuid logs_collector.LogsCollectorGuid
+}
+
+func newKubernetesLogsCollectorObjectAttributesProvider(logsCollectorGuid logs_collector.LogsCollectorGuid) *kubernetesLogsCollectorObjectAttributesProviderImpl {
+	return &kubernetesLogsCollectorObjectAttributesProviderImpl{
+		logsCollectorGuid: logsCollectorGuid,
+	}
+}
+
+func (provider *kubernetesLogsCollectorObjectAttributesProviderImpl) ForLogsCollectorDaemonSet() (KubernetesObjectAttributes, error) {
+	name, err := getCompositeKubernetesObjectName([]string{logsCollectorNamePrefix, string(provider.logsCollectorGuid)})
+	if err != nil {
+		return nil, err
```

Comment:
looks like we're not stacktrace.Propagate'ing here and below. Intentional?

Reply:
intentional laziness

Reply:
jk that err already comes wrapped w a detailed propagate from `getCompositeKubernetesObjectName` so i didnt see value in wrapping again at this layer